### DISCUSSION TITLE
Resource type

### DISCUSSION THREAD

**Discussion by zipus** [frePPLe/frepple#573] [Q&A] [answered]
Following my production example from: https://github.com/frePPLe/frepple/discussions/564#discussion-4492859
My resources `resource drill`, `painting room` are created with`'type': 'buckets_week'` and `'maximum': '40'`.
But once planned, when viewing the resource > detail plan I have multiple orders in the same start time and end time. What is causing this? The drill or the room can only have one operation at a time.
I'm wondering if I'm being hit by [Detailed scheduling of the operations within the time bucket isn’t considered useful in this type of plan.](https://frepple.com/docs/6.24/model-reference/resources.php)
Which is the recommended type for the use case?
Also, I can be totally wrong pointing this direction, and the problem is somewhere else, so any help will be appreciated.
Thanks.

**Comment by jdetaeye** [comment]
Check out the first paragraph on https://frepple.com/docs/current/examples/resource/resource-type.php to understand which resource type matches your expectations

**Comment by zipus** [comment]
It's not clear for me. As seams that the correct is default, but default can not have calendars.
Let's make it easy, which is the resource type on an employee that work 30h/week and can only do 1 task/1unit at each time?
Also, the 30h limitation should be managed by calendar, as can be working more or less on certain days.

**Comment by jdetaeye** [comment, marked as answer]
>... but default can not have calendars.

That's absolutely not true. You can attach 1) a size-max-calendar to let the capacity vary over time and 2) an availability-calendar to set the working hours.

> Let's make it easy, which is the resource type on an employee that work 30h/week and can only do 1 task/1unit at each time?

"can only do 1 task/1unit at each time"  -> default type, as described on  https://frepple.com/docs/current/examples/resource/resource-type.php

**Reply by zipus** [reply]
> That's absolutely not true. You can attach 1) a size-max-calendar to let the capacity vary over time and 2) an availability-calendar to set the working hours.

So the example for Time buckets was confusing me:
> E.g. A resource that has 40 hours available per week.

As my default resource can also have the same example, we need to find a better example for time buckets.
